Sources/StreamVideo/OpenApi/generated/Models/OwnUserResponse.swift

```diff
@@ -1,69 +1,178 @@
 //
-// OwnUserResponse.swift
-//
-// Generated by openapi-generator
-// https://openapi-generator.tech
+// Copyright © 2024 Stream.io Inc. All rights reserved.
 //
 
 import Foundation
-
-
-public struct OwnUserResponse: Codable, JSONEncodable, Hashable {
+    
+public final class OwnUserResponse: @unchecked Sendable, Codable, JSONEncodable, Hashable {
```

Comment:
This one is a bit tricky. It seems like it's taken from chat and some of the chat things are non-optional. For example:
```
public var totalUnreadCount: Int
public var unreadChannels: Int
public var unreadCount: Int
public var unreadThreads: Int
```
So if `me` is returned for video, it will break the parsing.

Reply:
If it is non-optional, it should be provided. But, we can test this I guess.

Reply:
The problem is that those are chat related.

Reply:
@martinmitrevski I addressed this. Please take another look.